### PR TITLE
use ansible_local for lxc as well

### PR DESCRIPTION
**Pull request by chregu**
should work since vagrant 1.8.4

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -148,8 +125,8 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     end
 
     config.vm.provider "lxc" do |lxc, override|
-        override.vm.box = custom_config.get('lxc_box_name', 'jessie64-lxc')
-        box_url = custom_config.get('lxc_box_url', 'http://vagrantbox-public.liip.ch/liip-jessie64-lxc.box')
+        override.vm.box = custom_config.get('lxc_box_name', 'drifter/jessie64-base')
```
Comment (krtek4) [inline Vagrantfile]: not sure you need to override the box config if it's the same as the "normal" one since all info are in the json.

I'll test without :P

File: Vagrantfile.dist [hunk 2]
```diff
@@ -30,10 +30,10 @@ class CustomConfig
...
     config = YAML::load(File.open(parameters_file))
 
-    @lxc_box_url = config['lxc_box_url'] || "http://vagrantbox-public.liip.ch/liip-jessie64-lxc.box"
-    @lxc_box_name = config['lxc_box_name'] || 'jessie64-lxc'
-    @vbox_box_url = config['vbox_box_url'] || 'http://vagrantbox-public.liip.ch/liip-jessie64.box'
-    @vbox_box_name = config['vbox_box_name'] || 'jessie64'
+    @lxc_box_url = config['lxc_box_url']     || config['box_url']  || 'https://vagrantbox-public.liip.ch/drifter-jessie64-base.json'
+    @lxc_box_name = config['lxc_box_name']   || config['box_name'] || 'drifter/jessie64-base'
+    @vbox_box_url = config['vbox_box_url']   || config['box_url']  || 'https://vagrantbox-public.liip.ch/drifter-jessie64-base.json'
+    @vbox_box_name = config['vbox_box_name'] || config['box_name'] || 'drifter/jessie64-base'
```
Comment (krtek4) [inline Vagrantfile.dist]: :inception:

Comment (chregu) [inline Vagrantfile.dist]: We can of course discuss, what the default should be ;)

File: Vagrantfile (hunk 1 above)
Comment (krtek4) [inline Vagrantfile]: Ok, it's because of the config option... forget my comment :P

File: Vagrantfile.dist (hunk 2 above)
Comment (krtek4) [inline Vagrantfile.dist]: I don't remember exactly why we did it like that but if I am not mistaken if redundant with the `custom_config.get` method which also has a default now.

Also, with the new `json`  format for the boxes, I think we could drop all that, release v2, and inform about the BC :P

But let's do that in another PR or never ^^
